Makefile

```diff
@@ -91,17 +96,15 @@ endif
 package: build-if-changed
 	@echo "Bundling app and server"
 	@rm -rf $(BUILD_DIR)/public_desktop $(BUILD_DIR)/calypso
-	@NODE_PATH=calypso/client $(WEBPACK_BIN) --config $(THIS_DIR)/webpack.config.js
+	@NODE_PATH=calypso/server$(SEPARATOR)calypso/client $(WEBPACK_BIN) --config $(THIS_DIR)/webpack.config.js
```

Comment:
We probably should move `NODE_PATH` handling to Webpack config. @gwwar @blowery @aduth could you help with this?